akka-stream/src/main/scala/akka/stream/scaladsl/Source.scala

```diff
@@ -411,6 +411,12 @@ object Source {
       combineRest(2, rest.iterator)
     })
 
+  def transpose[T, U](sources: Seq[Source[T, _]]): Source[Seq[T], NotUsed] = sources match {
```

Comment:
As we already have `zip` and `zipWith` it think it is a bad idea to introduce a new stage that does not follow that naming, the bacon.js naming mentioned in the ticket resonates well with me, I think it should be `zipAsSeq`